templating.rst

```diff
@@ -180,22 +184,14 @@ First, build a base layout file:
         <!DOCTYPE html>
         <html>
             <head>
-                <meta charset="UTF-8">
-                <title>{% block title %}Test Application{% endblock %}</title>
+                <meta charset="UTF-8" />
+                <title>{% block title %}Welcome!{% endblock %}</title>
+                {% block stylesheets %}{% endblock %}
+                <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
             </head>
             <body>
-                <div id="sidebar">
-                    {% block sidebar %}
-                        <ul>
-                            <li><a href="/">Home</a></li>
-                            <li><a href="/blog">Blog</a></li>
-                        </ul>
-                    {% endblock %}
-                </div>
-
-                <div id="content">
-                    {% block body %}{% endblock %}
-                </div>
+                {% block body %}{% endblock %}
+                {% block javascripts %}{% endblock %}
             </body>
         </html>
```

Comment:
I replaced with the REAL `base.html.twig`... because users actually start with this :)